### ISSUE TITLE
Signature should read "Government of Canada" or "Gouvernement du Canada" using either English or French

### ISSUE THREAD

**Issue by adriannelee** [cds-snc/design-gc-conception#2089]
All Pages
https://fable-test.d2fzvkr34ak02t.amplifyapp.com/view-holidays/nationwide/
2025-12-05
Fail
**Description of Issue (required):**

The Government of Canada logo on the top of every page, has a title that is in two languages (French and English): "Government of Canada / Gouvernement du Canada".

On the English language version of the page however, the French text does not have a French language attribute.

**Impact on Users:**

Screen readers use language attributes to know how to pronounce text correctly. If a webpage doesn’t specify the language of the content, the screen reader will guess based on the user’s default language. This can lead to mispronunciations or confusion if the actual text is in a different language.

**Sample Code** (required):

<svg version="1.1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 819 78" preserveAspectRatio="xMinYMin meet" role="img" aria-labelledby="signature-en-title">…</svg>
**Recommendation (required if not SAMS/OCMC):**

We recommend ensuring that in all cases where text strings involve languages other than the page language, language attributes are used to inform screen readers of the language the text strings are in.

**Additional Resources:**

https://www.w3.org/WAI/WCAG21/Understanding/language-of-parts.html
**Steps to Reproduce:**

1. Open the solution: https://fable-test.d2fzvkr34ak02t.amplifyapp.com/

2. Inspect the Government of Canada logo on the top of the page.

**Comment by SmartMouthWords** [comment]
@ethanWallace Did you want a content or localization review on this?

**Comment by ethanWallace** [comment]
@SmartMouthWords No it's all good. This issue is more around the formatting of the HTML elements on the code side.
